types/emscripten/index.d.ts

```diff
@@ -114,9 +117,51 @@ declare namespace FS {
         node: FSNode;
     }
 
-    interface FSStream {}
-    interface FSNode {}
-    interface ErrnoError {}
+    interface Mount {
+        type: Emscripten.FileSystemType;
+        opts: object;
+        mountpoint: string;
+        mounts: Mount[];
+        root: FSNode;
+    }
+
+    class FSStream {
+        shared?: {
+            flags: number;
+            position: number;
+        };
+        node: FSNode;
```

Comment:
It looks like this is initialized to `null`. Also not clear if it’s supposed to be public API since `object` gets and sets it.

Reply:
Does 0066922 fix it?

Reply:
No, the question mark modifier introduces `undefined`, not `null`. `node: FSNode | null` is what I would expect (if you are set on keeping the field).

Reply:
Removed the field in a2b5ba1